source/sonata_simulation.rst

```diff
@@ -280,6 +281,7 @@ Cells are held at indicated membrane voltage by injecting adapting current.
    Property                       Type       Requirement  Description
    ============================== ========== ============ ==========================================
    voltage                        float      Mandatory    Specifies the membrane voltage the targeted cells should be held at in mV.
+   rs                             float      Optional     Specifies the series resistance in M :math:`\Omega`. Default is 0.01 M :math:`\Omega`.
```

Comment:
would it make sense to call this `resistance` or `series_resistance`?

Reply:
`series_resistance` would definitely be more human-readable